packages/plugin-conversation/src/share-activity.js

```diff
@@ -78,6 +78,20 @@ const ShareActivity = SparkPlugin.extend({
    * @returns {EventEmittingPromise}
    */
   add(file) {
+    let actions;
+
+    // ArrayBuffer might be located in an object with the actions.
+    if (has(file, `file`) && has(file, `actions`)) {
```

Comment:
That is a great question.  My view was that making an exception was going to be safer than changing the API.  The consequences of the change the API would be big.  An alternative would be to add a new function called `addAsOptions({file: blob/arraybuffer, actions: {})`.  Is that closer to what you are thinking?